modules/core/src/main/java/org/apache/ignite/internal/processors/failure/FailureProcessor.java

```diff
@@ -39,6 +40,13 @@
     private static final boolean IGNITE_DUMP_THREADS_ON_FAILURE =
         IgniteSystemProperties.getBoolean(IgniteSystemProperties.IGNITE_DUMP_THREADS_ON_FAILURE, true);
 
+    /** Ignored failure log message. */
+    public static final String IGNORED_FAILURE_LOG_MSG = "Possible failure suppressed accordingly to a configured handler ";
```

Comment:
Once again. Make both constants  package-local and move your test class to the same package.